godot-bindings/Cargo.toml

```diff
@@ -37,18 +37,16 @@ api-custom = ["dep:bindgen", "dep:regex", "dep:which"]
 api-custom-extheader = []
 
 [dependencies]
-gdextension-api = { version = "0.2.2", git = "https://github.com/godot-rust/godot4-prebuilt", branch = "releases" }
+gdextension-api = { workspace = true }
 
 # Do not use bindgen 0.69, it contains regression that forces recompilation of code.
```

Comment:
This comment no longer makes sense (dependency is specified in main cargo.toml + we are way past 0.69 now  I guess)

Reply:
Thanks! Removed that line here, but added a comment in the main `Cargo.toml`.

In fact, we used bindgen 0.68 before this PR, due to this reason. I _hope_ this problem is fixed now, so I'll leave the comment in case it shows up again.